src/scripts/core/models/token.ts

```diff
@@ -6,31 +6,54 @@ import { gsap } from 'gsap';
 
 export class Token extends Container {
 
-    public matched: boolean;
-    private _parentID: number;
-    private parentSize: number;
-    public _verticalIndex: number;
-    private availHeight: number;
+    public matched: boolean = false;
+    private _parentID: number = 0;
+    private parentSize: number = 0;
```

Comment:
Some private variables are prefixed with `_` and some aren't. This should be kept consistent which ever way you decide to do it.